pkg/sql/tablemetadatacache/cluster_settings.go

```diff
@@ -34,3 +39,11 @@ var DataValidDurationSetting = settings.RegisterDurationSetting(
 		return nil
 	}),
 	settings.WithPublic)
+
+var updateJobBatchSizeSetting = settings.RegisterIntSetting(
```

Comment:
Are we intentionally not making this public?

Reply:
Hmm good question let's ask Kevin if we want to expose this. I'll leave it as private for now.